# 발행 주기 설정 (Hz)
publish_rate: 100

# 선택적 타임스탬프 참조 토픽 설정
timestamp_topic:
  name: "/livox/lidar"
  type: "sensor_msgs/msg/PointCloud2"

transforms:
  - frame_id: "base_link"
    child_frame_id: "livox_frame"
    timestamp_topic:
      name: "/livox/lidar"
      type: "sensor_msgs/msg/PointCloud2"
    translation:
      x: 0.21
      y: 0.0
      z: 0.71
    rotation:
      roll: 0.0
      pitch: 0.0
      yaw: 0.0

  - frame_id: "base_link"
    child_frame_id: "zed_camera_link"
    timestamp_topic:
      name: "/zed/zed_node/rgb/image_rect_color"
      type: "sensor_msgs/msg/Image"
    translation:
      x: 0.25
      y: 0.0
      z: 0.58
    rotation:
      roll: 0.0
      pitch: 0.0
      yaw: 0.0 